processing/src/main/java/io/druid/query/lookup/LookupBean.java

```diff
@@ -19,32 +19,76 @@
 
 package io.druid.query.lookup;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
 
 //TODO merge this code to the same definition when pr/1576 is merged
```

Comment:
sorry for the confusion but i think there was some duplicate code for this class between 2 PRs, but can't find out if it was fixed or not a the top of my head.